GREEDY/ReceiptCreatings/ReceiptCreating.cs

```diff
@@ -33,12 +34,14 @@ public Receipt FullReceiptCreating(Bitmap image)
             var linesOfText = _ocr.ConvertImage(image);
             var date = GetDateForReceipt(linesOfText);
             var shop = GetShopFromData(linesOfText);
+            //var percentageMatched = float.Parse(linesOfText.Last(), CultureInfo.InvariantCulture.NumberFormat);
```

Comment:
commented lines (atleast write above why is it commented and why it still may be needed)